graphics.tex

```diff
@@ -132,17 +129,188 @@ \subsection*{Hires}
+
+The next question is, how can we control specific bits in a register? If you turn on your computer and enter the following command
+
+\begin{screenoutput}
+PRINT PEEK($D031)
+ 224
+\end{screenoutput}
+
+you get back a decimal value like 224. The decimal value does not help, you need to convert the number into its binary representation. 
+
+\begin{tabular}{|l|l|l|l|l|l|l|l|l|}
+	\hline
+	Bit 7 & Bit 6 & Bit 5 & Bit 4 & Bit 3 & Bit 2 & Bit 1 & Bit 0 & Decimal value \\
+	H640 & FAST & ATTR & BPM & V400 & H1280 & MONO & INT & ~ \\
+	\hline
+    1 & 1 & 1 & 0 & 0 & 0 & 0 & 0 & 224 \\
+    0 & 1 & 1 & 0 & 0 & 0 & 0 & 0 & 96 \\    
+	\hline
+\end{tabular}
+
+The first line in the table shows the binary number for decimal 224, this should be the case when you turn on your MEGA65. Databit 7, H640-mode is activated as expected. In the second line you see the value you have to use when you want to reduce down to 320x200: databit 7 is set to 0, so H640-mode is turned off.
+
+\begin{screenoutput}
+POKE $D031,96
+\end{screenoutput}
+
+But wait, poking the value directly is not always the correct way. If you know exactly what you do or better which state each bit of the register should have, can you continue like this. Storing the value 96 means setting all the databits. This is not always what we want. In some cases we might manipulate one specific bit, for instance databit 7 for the resolution while keeping the other bits in their existing state. You can achieve this with logical operations using the AND or OR operator.
+
+\begin{tabular}{ ccc }   
+	~ & ~ & ~ \\  
+	% first table (AND)
+	\begin{tabular}{ |c|c| } 
+		\hline
+		Operation & Result \\
+		\hline
+		0 AND 0 & 0  \\
+		1 AND 0 & 0  \\
+		0 AND 1 & 0  \\
+		1 AND 1 & 1  \\						
+		\hline
+	\end{tabular} &  
+    % some space between the two tables
+	\begin{tabular}{c}
+	    \hspace{1cm} \\
+	\end{tabular}
+	% second table (OR)
+	\begin{tabular}{ |c|c| } 
+   	    \hline
+        Operation & Result \\
+		\hline
+		0 OR 0 & 0  \\
+		1 OR 0 & 1  \\
+		0 OR 1 & 1  \\
+		1 OR 1 & 1  \\						
+		\hline	
+	\end{tabular} \\
+\end{tabular}
+
+The table above is important because it shows, how the bits will be modified if you put an operator on it. If you think about the idea of logical operations, you will see that you can use AND to set one specific bit and OR to get the exact opposite result. To be honest, this method can not only be used to set a single bit, but also to change any number of bits but to keep things simple we modify only one. Back to the "bad" poke of 96 what we've seen above, it could be better written like this:
+
+\begin{screenoutput}
+POKE $D031, PEEK($D031) AND 127 :REM SET BIT 7
+POKE $D031, PEEK($D031) OR 128  :REM UNSET BIT 7
+\end{screenoutput}
+
+So that you understand this operations, again you need to convert the decimal numbers into its binary format. Next we apply the AND / OR operations bit by bit according to the rules in the table above to it. 
```

Comment:
`these` operations

Change `the decimal numbers` to this `each decimal number`, it reads better that way for me.

Change `in the table above to it.` to `in the prior logic tables.`.